### PR TITLE
Updated README.md according to previous changes of docker files.

### PR DESCRIPTION
**Pull request by jayeshjeh**
### Description
README.md has been updated according to the new changes pushed into the /examples/log-ingestion/ in last commit. 
One can follow and understand better. 
 
### Issues Resolved
Resolves #[Issue number to be closed when this PR is merged]
 
### Check List
- [ ] New functionality includes testing.
- [ ] New functionality has a documentation issue. Please link to it in this PR.
  - [ ] New functionality has javadoc added
- [x] Commits are signed with a real name per the DCO 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/data-prepper/blob/main/CONTRIBUTING.md).

### CODE REVIEW

File: examples/log-ingestion/README.md [hunk 1]
```diff
@@ -29,35 +29,38 @@ FluentBit is tailing to collect logs from.
...
+1. Take a look at [log_pipeline.yaml](log_pipeline.yaml). This configuration will take logs sent to the [http source](../../data-prepper-plugins/http-source), 
 process them with the [Grok Processor](../../data-prepper-plugins/grok-prepper) by matching against the `COMMONAPACHELOG` pattern, 
 and send the processed logs to a local [OpenSearch sink](../../data-prepper-plugins/opensearch) to an index named `apache_logs`.
 
+2. And [data-prepper-config.yaml](data-prepper-config.yaml) is also mounted in [docker-compose-dataprepper.yaml](docker-compose-dataprepper.yaml) which will help us to configure our data-prepper server. 
 
-3. Run the Data Prepper docker image with the `log_pipeline.yaml` from step 2 passed in. This command attaches the Data Prepper Docker image to the Docker network `log-ingestion_opensearch_net` so that 
-FluentBit is able to send logs to the http source of Data Prepper.
+
+3. Run the Data Prepper docker compose file where we are using `log_pipeline.yaml`. Now fluentBit is able to send logs to the http source of Data Prepper.
```
Comment (KarstenSchnitter) [inline examples/log-ingestion/README.md:47]:
```suggestion
3. Run the Data Prepper docker compose file where we are using `log_pipeline.yaml`. Now FluentBit is able to send logs to the http source of Data Prepper.
```

**Review by KarstenSchnitter — COMMENTED** [review]
Thanks for providing this change. I found just one minor typo. I did not try out the procedure to compare with the documentation though.

File: examples/log-ingestion/README.md (hunk 1 above)
Comment (jayeshjeh) [reply examples/log-ingestion/README.md]: Hi @KarstenSchnitter changes are made.

**Review by KarstenSchnitter — APPROVED** [review]
Looks good to me now.

**Pull request merged**